hackclub.com.yaml

```diff
@@ -2822,4 +2822,3 @@ internal:
 - ttl: 600
   type: A
   value: 104.131.182.116
-  
```

Comment:
Could you revert your changes to this file entirely?

Reply:
how do i do this?